src/promptflow/tests/sdk_cli_azure_test/unittests/test_azure_cli_time_consume.py

```diff
@@ -0,0 +1,103 @@
+import cProfile
+import contextlib
+import io
+import multiprocessing
+import pstats
+import sys
+from unittest import mock
+import pytest
+from promptflow._cli._pf_azure.entry import main
+from promptflow.azure.operations._run_operations import RunOperations
+
+FLOWS_DIR = "./tests/test_configs/flows"
+DATAS_DIR = "./tests/test_configs/datas"
+
+
+def run_cli_command(cmd, time_limit=3600, is_print_stats=True, result_queue=None):
+    with mock.patch.object(RunOperations, "create_or_update") as create_or_update_fun, \
+            mock.patch.object(RunOperations, "update") as update_fun, \
+            mock.patch.object(RunOperations, "get") as get_fun, \
+            mock.patch.object(RunOperations, "restore") as restore_fun:
+        create_or_update_fun.return_value._to_dict.return_value = {"name": "test_run"}
+        update_fun.return_value._to_dict.return_value = {"name": "test_run"}
+        get_fun.return_value._to_dict.return_value = {"name": "test_run"}
+        restore_fun.return_value._to_dict.return_value = {"name": "test_run"}
+
+        sys.argv = list(cmd)
+        with cProfile.Profile() as pr:
```

Comment:
no need to run cProfile during tests. only do it when local profiling

Reply:
done